src/components/grid/PCol.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div v-bind:class="[`col-${sizeXs}`,`col-sm-${sizeSm}`,`col-md-${sizeMd}`,`col-lg-${sizeLg}`,`col-xl-${sizeXl}`]">
```

Comment:
`v-bind:class`

`:class`

Reply:
Also please make a computed property called `classNames`.

If there is no `sizeXs` **don't** add `col-xs-undefined`.

Reply:
All class names should start with `puto-`